lib/scrollable_modal_webview.dart

```diff
@@ -10,22 +10,81 @@ void showScrollableModalWebView({
   required BuildContext context,
   required WebViewController controller,
   required String url,
-  PreferredSizeWidget? header,
+  Widget? header,
   bool scrollable = true,
   double initialChildSize = 1.0,
-  ShapeBorder? shape,
+  BorderRadiusGeometry? borderRadius,
+  Color? headerColor,
 }) {
   if (!(Platform.isIOS || Platform.isAndroid)) {
     throw Exception('This OS is not supported');
   }
   showModalBottomSheet(
     context: context,
     isScrollControlled: true,
-    shape: shape,
-    builder: (BuildContext context) => const ScrollableModalWebView(),
+    shape: RoundedRectangleBorder(
+        borderRadius: borderRadius ??
+            const BorderRadius.vertical(top: Radius.circular(0))),
+    builder: (BuildContext context) => ScrollableModalBottomSheet(
+      controller: controller,
+      initialChildSize: initialChildSize,
+      scrollable: scrollable,
+      url: url,
+      header: header,
+      headerColor: headerColor,
+      borderRadius: borderRadius,
+    ),
   );
 }
 
+class ScrollableModalBottomSheet extends StatelessWidget {
+  final WebViewController controller;
+  final Widget? header;
+  final double initialChildSize;
+  final bool scrollable;
+  final String url;
+  final Color? headerColor;
+  final BorderRadiusGeometry? borderRadius;
+  const ScrollableModalBottomSheet(
+      {Key? key,
+      required this.controller,
+      this.header,
+      required this.initialChildSize,
+      required this.scrollable,
+      required this.url,
+      this.headerColor,
+      this.borderRadius})
+      : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return DraggableScrollableSheet(
+        expand: false,
+        initialChildSize: initialChildSize,
+        builder: (context, scrollController) {
+          return Column(
+            children: [
+              if (header != null) ...{
+                Container(
+                  decoration: BoxDecoration(
+                    borderRadius: borderRadius,
+                    color: headerColor,
+                  ),
+                  height: MediaQuery.of(context).padding.bottom,
```

Comment:
```suggestion
                  height: MediaQuery.of(context).padding.top,
```
上部のsafe area分を開けたいのであればtopかなと思ったのですがいかがでしょうか？

Reply:
僕の環境だとtopにすると逆にsafeAreaが無くなります！

<img width="300" alt="スクリーンショット 2023-02-02 17 44 51" src="https://user-images.githubusercontent.com/85224998/216274785-98cae43a-0f6c-40d8-bed0-cc1f8dfeb7e1.png">

Reply:
ここをbottomとするならば、「topとしない」それなりの理由がないと僕以外の人から質問きた時に納得させることが困難かと思われますが、そこのところ回答は用意できますでしょうか？

Reply:
topにして解決できないのは別の部分に原因があるのではと考えています。

Reply:
https://github.com/flutter/flutter/issues/103585
ここら辺のディスカッションとか参考になりそうな気がします。

Reply:
完全に真逆なので、確かに他の人からの質問に納得する答えが返すのは難しいですね。後ろのタスクが詰まっているので、このsafeArea問題は一旦保留にして、また別のPRで対応したいと思います！

Reply:
オッケーです。
https://github.com/shinonome-inc/scrollable_modal_webview/pull/5/commits/4447fccf0373eff8ff2c515ec4c4db89973b3b04
↑こちらのコミットrevertしてもらって、コメント打つか？
それか該当箇所にコメントアウトしてからマージしたいですね。